.github/workflows/test.yml

```diff
@@ -55,7 +55,7 @@ jobs:
           export GOMAXPROCS=1
           args=("-p" "1" "-race")
           export K6_BROWSER_HEADLESS=true
-          go test "${args[@]}" -timeout 5m ./...
+          go test "${args[@]}" -timeout 10m ./...
```

Comment:
Why do we need such a large timeout?

Reply:
Since the concurrency is 1 now, and it was 2 before, I thought it would make sense to double the timeout. When I tested it, tests took about 2x longer.